examples/mesh-gateways/datacenter/consul-server.tf

```diff
@@ -0,0 +1,42 @@
+# Run the Consul dev server as an ECS task.
+module "dev_consul_server" {
+  name                        = "${var.name}-consul-server"
+  source                      = "../../../modules/dev-server"
+  datacenter                  = var.datacenter
+  primary_datacenter          = var.primary_datacenter
+  retry_join_wan              = var.retry_join_wan
+  primary_gateways            = var.primary_gateways
+  ecs_cluster_arn             = aws_ecs_cluster.this.arn
+  service_discovery_namespace = var.datacenter // TODO mesh gateway WAN fed requires the service discovery ns to be the datacenter? may need to remove this..
```

Comment:
I think just setting this to `datacenter` is fine since the dev server is only for testing.